scylla-cql/src/frame/frame_errors.rs

```diff
@@ -85,7 +83,24 @@ pub enum SetKeyspaceParseError {
 }
 
 /// An error type returned when deserialization of
-/// `RESULT::Schema_change` response fails.
+/// `EVENT` response fails.
+#[non_exhaustive]
+#[derive(Error, Debug)]
+pub enum CqlEventParseError {
+    #[error("Malformed event type string: {0}")]
+    EventTypeParseError(LowLevelDeserializationError),
+    #[error("Unknown event type: {0}")]
+    UnknownEventType(String),
+    #[error("Failed to deserialize schema change event: {0}")]
+    SchemaChangeEventParseError(#[from] SchemaChangeEventParseError),
+    #[error("Failed to deserialize topology change event: {0}")]
+    TopologyChangeEventParseError(ClusterChangeEventParseError),
+    #[error("Failed to deserialize status change event: {0}")]
+    StatusChangeEventParseError(ClusterChangeEventParseError),
```

Comment:
Ok, I see you already added this type. Then why is there still `SchemaChangeEventParseError` variant in `CqlResultParseError`?

Reply:
And yes, it may be that schema change event has the same binary form as schema change response - but they are different things anyway and we should not mix them up.

Reply:
I changed the variant name to `SchemaChangeParseError`